Tools/autotest/arduplane.py

```diff
@@ -2942,8 +2942,6 @@ def TerrainLoiter(self):
             gpi = self.assert_receive_message('GLOBAL_POSITION_INT')
             terrain = self.assert_receive_message('TERRAIN_REPORT')
             rel_alt = terrain.current_height
-            self.progress("%um above terrain (%um bove home)" %
-                          (rel_alt, gpi.relative_alt/1000.0))
```

Comment:
Why is this being removed?

Too noisy?

Reply:
I thought I had put it in myself. I was trying to remove debug code I had added. I'll put it back.

Reply:
Reverted

Reply:
Note quite; we print the final value just before dying so you know what the bad values were

Reply:
Got it this time I think.